app/templates/content.hbs

```diff
@@ -1,9 +1,12 @@
-{{title model.title}}
+{{title node.title}}
 
 <div id="view-page">
     <div class="p-t-xl m-t-xl p-b-md m-b-md dark-overlay-header-background header-row">{{!HEADER ROW}}
         <div class="container">{{!CONTAINER DIV}}
-            <h1 id="preprintTitle" class="p-b-md">{{model.title}}</h1>
+            <h1 id="preprintTitle" class="p-b-md">{{node.title}}</h1>
+            {{#if isAdmin}}
+                <a type="button" class="pull-right btn btn-success m-h-lg" href={{concat model.links.html '?edit'}}  {{action "click" 'link' "Content - Edit Preprint" (concat model.links.html '?edit')}}>{{t "content.project_button.edit_preprint"}}</a>
```

Comment:
@binoculars I had tried to use a link-to helper initially, but couldn't figure out how to make it work with the weird routing.

The issue is that we are on the content page here, and "edit" template also uses the content route.  So if it was something like this... `                {{#link-to "content" (query-params edit="true") class="pull-right btn btn-success m-h-lg" }}{{t "content.project_button.edit_preprint"}}{{/link-to}}, this button would just tack ?edit=true to the url but wouldn't reload the models and controllers, because we are already on the content page.  I need to force a reload so the controller/template for the submit page will be loaded instead.